precompiles/assets-erc20/src/lib.rs

```diff
@@ -387,7 +387,9 @@ where
 				delegate: Runtime::Lookup::unlookup(spender),
 				amount,
 			},
-		)
+		)?;
```

Comment:
good one

Reply:
This is because I improved the util for Substrate calls to return the dispatch info in case of success, and return a new enum in case of failure that allow to get the DispatchError as is. Currently I have not used it, but we could match on the module and error indices to recognize the error and have clearer revert messages.

Due to the needed type convertion and not wanting to deal with the dispatch info adding `?; Ok(())` is shorter.